src/ts_catalog/continuous_agg.c

```diff
@@ -1184,6 +1185,18 @@ drop_continuous_agg(FormData_continuous_agg *cadata, bool drop_user_view)
 
 	if (OidIsValid(mat_hypertable.objectId))
 	{
+#if PG14_GE
+		OsmCallbacks *callbacks = ts_get_osm_callbacks();
+
+		if (callbacks)
+		{
+			Hypertable *mat_ht = ts_hypertable_get_by_id(cadata->mat_hypertable_id);
+
+			callbacks->hypertable_drop_hook(NameStr(mat_ht->fd.schema_name),
```

Comment:
this file is called `contious_agg.c` - these lines looks out-of-place here ; I think this hook should be invoked during `ts_hypertable_delete_by_id` or not?

Reply:
Would you say `hypertable_tuple_delete` is a good place for it?